adapter/src/index.js

```diff
@@ -23,6 +23,7 @@ const AppAdapter = ({
     loginApp,
     children,
 }) => {
+    const [loginBaseUrl, setLoginBaseUrl] = useState(url)
```

Comment:
We need to pass the base url (logic in ServerVersionProvider) to LoginAppWrapper in order to display the correct redirection for safe mode login. 

The easiest option would be to just move the `LoginAppWrapper` component into `ServerVersionProvider`, but I thought maybe the current code structure was preferable in terms of displaying where the wrapper was? So hence this use of state variable to track the updates from `ServerVersionProvider` in this parent component.

Reply:
Wait, why can't we just read from the server-version context in `LoginAppWrapper`, since it's a child of that provider?

Reply:
@Birkbjo : we can use `useConfig` within `LoginAppWrapper` to get the baseUrl. I didn't want to do that initially because I thought it was kind of confusing to use the hook from app-runtime there, but we're also using `useConfig` in the `ConnectedHeaderBar` component, so I guess using `useConfig` in this case would be more consistent. I think my hesitation was just that within the directory we have some components that have access to the context and some that do not, so you have to pay attention to where these components are in the hierarchy.

I've updated to use `useConfig` in `LoginAppWrapper`. 🙏